server/pkg/webapp/serve.go

```diff
@@ -183,3 +239,29 @@ func replaceBaseHrefPlaceholder(buf []byte, app, baseHref string) []byte {
 
 	return fixed
 }
+
+func traverseScriptsNode(n *html.Node, scripts *[]scriptAttrs) {
```

Comment:
Nooo; utilize returns over this pointer stuff; recursive functions can still return correctly